### DISCUSSION TITLE
Directory Trustwalletcore not found

### DISCUSSION THREAD

**Discussion by nexiz578** [trustwallet/wallet-core#3298] [Q&A]
TrustWalletCore library not found.  Please provide TrustWalletCore build
  directory with -DWALLET_CORE.  Example: cmake .
How to fix it?

**Comment by Milerius** [comment]
For example: `-DWALLET_CORE=../../../wallet-core`

**Reply by nexiz578** [reply]
CMake Warning:
  No source or binary directory provided.  Both will be assumed to be the
  same as the current working directory, but note that this warning will
  become a fatal error in future CMake releases.


CMake Error at CMakeLists.txt:37 (message):
  TrustWalletCore library not found.  Please provide TrustWalletCore build
  directory with -DWALLET_CORE.  Example: cmake .
  -DWALLET_CORE=../wallet-core
